test/tests/functional/pbs_que_resc_usage.py

```diff
@@ -71,31 +68,30 @@ def test_resc_assigned_set_unset(self):
         """
 
         # set a resource and unset it without using
-        a = {'resources_available.ncpus': 4}
-        self.server.manager(MGR_CMD_SET, QUEUE, a, id='workq')
+        self.server.manager(MGR_CMD_SET, QUEUE, {
+                            'resources_available.ncpus': 4}, id='workq')
         q_status = self.server.status(QUEUE, id='workq')
         self.assertEqual(
             int(q_status[0]['resources_available.ncpus']), 4, self.err_msg)
         self.assertNotIn('resources_assigned.ncpus',
-                         q_status[0].keys(), self.err_msg)
+                         q_status[0], self.err_msg)
         self.server.manager(MGR_CMD_UNSET, QUEUE,
                             'resources_available.ncpus', id='workq')
         q_status = self.server.status(QUEUE, id='workq')
         self.assertNotIn('resources_available.ncpus',
-                         q_status[0].keys(), self.err_msg)
+                         q_status[0], self.err_msg)
 
         # set the resource and unset it after using
-        a = {'resources_available.ncpus': 8}
-        self.server.manager(MGR_CMD_SET, QUEUE, a, id='workq')
+        self.server.manager(MGR_CMD_SET, QUEUE, {
+                            'resources_available.ncpus': 8}, id='workq')
         q_status = self.server.status(QUEUE, id='workq')
         self.assertEqual(
             int(q_status[0]['resources_available.ncpus']), 8, self.err_msg)
         # job submission
-        j_attr = {ATTR_queue: 'workq', 'Resource_List.select': '1:ncpus=3'}
-        j1 = Job(TEST_USER, j_attr)
+        j1 = Job(attrs={'Resource_List.ncpus': '3'})
         j1.set_sleep_time(30)
         jid_1 = self.server.submit(j1)
-        j2 = Job(TEST_USER, j_attr)
+        j2 = Job(attrs={'Resource_List.ncpus': '3'})
```

Comment:
Since we're submitting the same job twice, setting the values once in a variable and using it twice is more generic.  If the test ever needs to be changed, we would change the value once.